api.md

```diff
@@ -2041,6 +2043,36 @@ SELECT * FROM show_tablespaces('conditions');
 
 ---
 
+## timescaledb_pre_restore() [](timescaledb_pre_restore)
+
+Perform the proper operations to allow restoring of the database via `pg_restore` to commence.
+Specifically this sets the `timescaledb.restoring` GUC to `on` and stops any
+background workers which may have been performing tasks until the restore
+completes. See [backup/restore docs][backup-restore] for more information.
+
+>:WARNING: After running `SELECT timescaledb_pre_restore()` you must run the 
+  `timescaledb_post_restore` before using the database normally.
```

Comment:
either remove "the" from the previous line, or add "command" after `timescaledb_post_restore`